Sources/Sentry/Public/SentryAttachment.h

```diff
@@ -8,6 +8,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Attachment Type
+ */
```

Comment:
`m`: I think it would make sense to explain a bit what these attachment types are and to explain clearly that the typical user doesn't need to worry about this as mostly HybridSDKs use it. The same applies for the init methods.